tests/desktop/consumer_pages/test_reviews.py

```diff
@@ -41,10 +41,6 @@ def _create_review(self, mozwebqa):
         details_page = add_review_box.write_a_review(mock_review['rating'], mock_review['body'])
 
         # Step 4 - Check review
-        details_page.wait_notification_box_visible()
```

Comment:
I assume that your testing indicates that this, and the following waits for notifications are unnecessary. I wonder why they were added in the first place. I would have assumed that they were needed to address timing issues for the test, but perhaps those issues have gone away.